client/core/core.go

```diff
@@ -120,6 +120,9 @@ func (dc *dexConnection) marketConfig(mktID string) *msgjson.Market {
 // marketMap creates a map of this DEX's *Market keyed by name/ID,
 // [base]_[quote].
 func (dc *dexConnection) marketMap() map[string]*Market {
+	if !dc.connected {
```

Comment:
On master `connected` is a uint32 atomic, so you'll have to do similarly to other places like `atomic.LoadUint32(&dc.connected) == 1`

Reply:
Thanks for the review. Now the CI passes!